src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/multi-cluster/multi-cluster-form/multi-cluster-form.component.html

```diff
@@ -110,6 +118,33 @@
             </span>
           </div>
         </div>
+        <div class="form-group row"
+             *ngIf="remoteClusterForm.getValue('showToken') && action !== 'edit'">
+          <label class="cd-col-form-label required"
+                 for="prometheusApiUrl"
+                 i18n>Prometheus API URL
+            <cd-helper>
+              <cd-alert-panel type="info"
+                              spacingClass="mb-3"
+                              size="normal"
+                              showTitle="false">
+              <p>Enter the Dashboard API URL. You can retrieve it from the CLI with:</p>
+                <cd-code-block [codes]="[prometheusApiUrlCmd]"></cd-code-block>
+              </cd-alert-panel>
+            </cd-helper>
+          </label>
+          <div class="cd-col-form-input">
+            <input id="prometheusApiUrl"
+                   name="prometheusApiUrl"
+                   class="form-control"
+                   type="text"
+                   formControlName="prometheusApiUrl">
+            <span class="invalid-feedback"
+                  *ngIf="remoteClusterForm.showError('prometheusApiUrl', frm, 'required')"
+                  i18n>This field is required.
+            </span>
+          </div>
+        </div>
```

Comment:
btw, we are going to disable the auth_with_token toggle later on right?

Reply:
Yes, we'l do it in a separate PR